### ISSUE TITLE
change the second page alert and show it to next page

### ISSUE THREAD

**Issue by rajibmitra** [rajibmitra/simple-survey#4]
right now the alert is getting popped up for the score. we need to change it and show it to next page and show the solution after that.

**Comment by ooHAoo** [comment]
Hi,
I would like to help with this but I'm a bit confused.
Could you please give a bit more explanation,
 thanks!

**Comment by rajibmitra** [comment]
Sure @ooHAoo  , basically when you go to second page , you have some questions , and if you select YES as response , you will get 10 points ,  if you select Partially as response , you will get 5 points and if you select NO as option , you will get 0 point , after that when you hit submit , you get a average of all the points for example : if your point is 50 for 5 questions , you will get a rating of 10 out of 10 .

When you hit submit  , you get popped up notification with that score only , the objective of this issue is to change the notification alert and show the score on next page with some valuable comments .
Hope this helps.

**Comment by ooHAoo** [comment]
So you want discard the alert on SecondPage and have it placed in ThirdPage instead and add valuable comments?

If I'm reading correctly, do you want that score to show after you hit submit on the ThirdPage? and was there any specific comments that you want to add? Or just 'Your score is: ...'

**Comment by rajibmitra** [comment]
yes @ooHAoo Remove the next button from second page and add it on third page. 
Yes it would be fine to add 'Your score is: ...'

**Comment by ooHAoo** [comment]
Sorry I keep asking so many questions but just want to make sure i get it right.
Do you mean remove the 'submit' button from the second page? leave the normal 'Next' page and take out the submit page so we can use its functionality in the third page?

**Comment by rajibmitra** [comment]
its fine , please ask questions for clarity , I meant remove the **next** button , and only keep the **submit** button , when you hit the **submit** button , user should get in to third page with the **score** and show the the thrid page options .

**Comment by ooHAoo** [comment]
so you want the scores of the second page to load onto the third page as an alert?

**Comment by rajibmitra** [comment]
yes , I want the scores of the second page to load onto the third page , but not as an alert, can we show on the third page itself  ? @ooHAoo

**Comment by rajibmitra** [comment]
@ooHAoo  how is the things going one , let me know if you need any further discussion or help !

**Comment by ooHAoo** [comment]
Hey, took a little break but I will be back to it in a bit :)
I think I understand what you want for this issue now, just gotta plan and figure it out!

**Comment by rajibmitra** [comment]
cool !

**Comment by ooHAoo** [comment]
@rajibmitra 
so there is a way to do this. However, after doing some research it would be a good a good idea to extract the javascript logic into its own .js file.
I'm not sure if you want to do that or not.

Unless you have another way of using a variable of page 2 in page 3 without extracting the javascript part?

**Comment by rajibmitra** [comment]
yes @ooHAoo  , thats a good idea , please go ahead and create a separate js file .
